packages/puppeteer-core/src/common/types.ts

```diff
@@ -17,6 +17,7 @@
 import {JSHandle} from './JSHandle.js';
 import {ElementHandle} from './ElementHandle.js';
 import {LazyArg} from './LazyArg.js';
+import type {LastArrayElement, Split} from 'type-fest';
```

Comment:
is it possible to implement two helpers are are needed instead of brining in the entire lib?

Reply:
done.